main.go

```diff
@@ -127,7 +127,7 @@ func main() {
 		if tools.IsTool(cmd) {
 			code, err := tools.RunTool(cmd, args[2:])
 			if err != nil {
-				log.Print(err.Error())
+				fmt.Println(err.Error())
```

Comment:
why this change? log will send in stdout, fmt will send in stderr - I do not want errors in stdout!

Reply:
Oops sorry I forgot about it. The problem I was having is that log prints date and time in front of the message. With the forst awk test the `assert_line` it was failing because it was never matching, i changed it just to go on with the tests and forgot